sourmash/commands.py

```diff
@@ -646,8 +646,10 @@ def gather(args):
 
     if found and args.output:
         fieldnames = ['intersect_bp', 'f_orig_query', 'f_match',
+                      'f_match_orig',
```

Comment:
would it be better to keep this column last, in case someone out there is doing some crazy/wonky CSV parsing? If it is the last field at least it wouldn't break in the most common case (split by comma, grab 4th field)

Reply:
hmm. you have a point, but I'd also like to just say that people not using the headers are Doing Bad Things. grr.